types/seneca/index.d.ts

```diff
@@ -1,9 +1,9 @@
-// Type definitions for seneca v2.1.0
+// Type definitions for seneca v3.7.0
 // Project: https://www.npmjs.com/package/seneca
 // Definitions by: Peter Snider <https://github.com/psnider>
 //                 Kevyn Bruyere <https://github.com/kevynb>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped/seneca
-// TypeScript Version: 2.3
+// TypeScript Version: 3.0.1
```

Comment:
Please don't change this unless necessary. This is for setting the minimum supported version, it's best to keep 2.3 supported.